### PR TITLE
feat(website): move tile from react to website package

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

- Move the tile component out of the react package to the website package
- Horizontally center the thumbnail image within the tile
- Adjust grid layout so that it is more responsive

https://user-images.githubusercontent.com/35579930/151404633-96aa84be-5836-4e5f-95c6-d11f55bb51fa.mp4

### Potential Breaking Changes

BREAKING CHANGE: Tile component is no longer available for usage outside of the website

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Review by aiheon — APPROVED** [review]
That's really nice!! :tada:  :grin:

File: packages/react/src/components/tile/tests/Tile.spec.tsx [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-
-        expect(screen.getByRole('link', {name: /my component/i})).toHaveAttribute('href', '/somewhere');
-    });
-
-    it('does not render a link if no href is provided', () => {
-        render(<Tile title="My tile" description="not a link" svgName="plasmaComponentBox" />);
-
-        expect(screen.queryByRole('link')).not.toBeInTheDocument();
-    });
-});
```
Comment (mikedidomizio) [inline packages/react/src/components/tile/tests/Tile.spec.tsx:39]: Are we re-adding the tests or decided we don't need them?

Comment (aiheon) [reply packages/react/src/components/tile/tests/Tile.spec.tsx:39]: We don't do tests for the website package :+1:

Comment (mikedidomizio) [reply packages/react/src/components/tile/tests/Tile.spec.tsx:39]: Okay, thanks!

**Review by mikedidomizio — APPROVED** [review]

**Review by GermainBergeron — APPROVED** [review]

File: packages/react/src/components/tile/tests/Tile.spec.tsx (hunk 1 above)
Comment (gdostie) [reply packages/react/src/components/tile/tests/Tile.spec.tsx:39]: We don't run any test for the plasma website 🤷

**Pull request merged**